data/layer-groups/zoning-map-index.json

```diff
@@ -0,0 +1,68 @@
+{
+  "id": "zoning-map-index",
+  "legend": {
+    "label": "Zoning Map Index",
+    "tooltip": "This layer displays an index of 126 zoning map sections that are part of the NYC Zoning Resolution. Each section is identified by a unique number and covers an area of 8,000 feet by 12,500 feet of the City. Click on a section to view and print the corresponding zoning map.",
+    "infolink": "https://s-media.nyc.gov/agencies/dcp/assets/files/AE/about-zoning-maps.pdf"
+  },
+  "layers": [
+    {
+      "style": {
+        "id": "zoning-map-index-line",
+        "type": "line",
+        "source": "supporting-zoning",
+        "source-layer": "zoning-map-index",
+        "paint": {
+          "line-width": {
+            "stops": [
+              [
+                11,
+                1
+              ],
+              [
+                12,
+                3
+              ]
+            ]
+          },
+          "line-color": "#000000"
+        }
+      }
+    },
+    {
+      "style": {
```

Comment:
Not sure if this is a problem here or in Zola but I'd guess here:

The labels aren't rendering for me at some zoom levels:
Missing 11a subplan and 11c labels:
![image](https://github.com/user-attachments/assets/c8561fd7-faf3-4093-a48c-3fe16461686c)

Zoom in a bit more, they show up:
![image](https://github.com/user-attachments/assets/b2272c27-0479-4a5e-b36a-eb1849853f58)

Reply:
I somewhat addressed the issue but there is still one problematic area, 11c. The name, Lake Success, is smack dab in the middle of the grid. Nothing much we can do about it unless we change the order of the layers.